machines/vcItem.ts

```diff
@@ -42,7 +42,7 @@ const model = createModel(
     storeVerifiableCredential: null as VerifiableCredential,
     requestId: '',
     isVerified: false,
-    isPinned: false,
+    isPinned: null as boolean,
```

Comment:
Can this work with the initial value as `false` just like before?

I got this in typescript compiler hence asking
> console.log(null as boolean); # evaluates to `null`

```
Conversion of type 'null' to type 'boolean' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
```

Reply:
initially I thought of when the vc is loading when ever we click on kebab popup this line is making the `isPinned: false,` is making the value false even we pin the card so I have change to null initially then assigning the value in the last when vc is loading. Now I think we can use as `false` just like before.